### PR TITLE
Channel test added.  File structure updated.

### PR DESCRIPTION
**Pull request by DavidDierickx**
Added test to ensure that each channel is individually maxed.  Also fixed file structure to hopefully prevent future issues with reference to functions in other folders from test folder.

### CODE REVIEW

**Comment by GenevieveBuckley** [comment]
> Hello @DavidDierickx! Thanks for updating this PR. We checked the lines you've touched for [PEP 8](https://www.python.org/dev/peps/pep-0008) issues...

Nice, you got pep8speaks working very easily :)

File: max_intensity_projection/requirements.txt [hunk 1]
```diff
@@ -0,0 +1,7 @@
+
+pillow==5.4.1
+jupyter
+scikit-image=0.14.0
```
Comment (GenevieveBuckley) [inline max_intensity_projection/requirements.txt]: Nitpick: You're mixing single and double equals signs here. Figure out which is correct (I think double equals is right) and be consistent.

File: max_intensity_projection/main.py [hunk 2]
```diff
@@ -1,28 +1,44 @@
...
+    n_channels = image.shape[-1]
+    image = np.moveaxis(image, -1, 0)
+    results = []
+    for channel_image in image:
+        max_intensity = np.max(channel_image, axis=0)
+        results.append(max_intensity)
+    projected_max_intensity = np.stack(results, axis=-1)
+    return projected_max_intensity
+
+
```
Comment (GenevieveBuckley) [inline max_intensity_projection/main.py]: Also a nitpick: there are a lot of blank lines here. Convention is that files should end with a single blank line, can you get rid of the extras?

**Comment by GenevieveBuckley** [comment]
The code looks good to me, I'd be happy for you to merge.

It does look like travis is not enabled properly, we can look at that on Thursday when we go over your bokeh slider plot.

File: max_intensity_projection/requirements.txt (hunk 1 above)
Comment (DavidDierickx) [reply max_intensity_projection/requirements.txt]: I assume it's double equals as I manually added the single equals sign myself.  Will update.

File: max_intensity_projection/main.py (hunk 2 above)
Comment (DavidDierickx) [reply max_intensity_projection/main.py]: I added them in to be able to scroll in notepad++ and forgot to remove them.  Will delete.

**Pull request merged**